### PR TITLE
Update dm icon

### LINKED ISSUE #11556: Update: dm
### Brand Name

dm

### Official Resources for Icon and Color

- SVG: https://www.dm.de/safari-pinned-tab.svg

### Additional Comments

dm uses a different version for its logo in which the two lower waves can still be distinguished when displayed in a single color

### PR DESCRIPTION
**Pull request by DerBraky**
Untitled

**Issue:** closes #11556

**Popularity metric:**
- Similarweb Global Rank: [3,445](https://www.similarweb.com/website/dm.de) (German)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Adapted from: https://www.dm-jobs.com/dmTECH

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4396,7 +4396,12 @@
         {
             "title": "dm",
             "hex": "002878",
-            "source": "https://www.dm.de"
+            "source": "https://www.dm-jobs.com/dmTECH",
```
Comment (LitoMore) [inline _data/simple-icons.json:4399]: This page is down. Please use the root instead.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (mondeja) [inline _data/simple-icons.json:4399]:
```suggestion
            "source": "https://dm-jobs.com",
```

**Review by mondeja — CHANGES_REQUESTED** [review]
There are differences between your version (red) and the source of the issue (black):

image

Maybe the source has been updated. Would you mind to redo it?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4396,7 +4396,12 @@
         {
             "title": "dm",
             "hex": "002878",
-            "source": "https://www.dm.de"
+            "source": "https://www.dmtech.tech",
```
Comment (adamrusted) [inline _data/simple-icons.json:4399]: The main website still applies - dmtech is just the technology branch of the business.

```suggestion
            "source": "https://www.dm.de",
```

File: icons/dm.svg
Comment (adamrusted) [inline icons/dm.svg:1]: Seeing some differences between your icon (black) and the icon from the website header (red). Could you take another look please @DerBraky?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Review by LitoMore — APPROVED** [review]

**Pull request merged**